### PR TITLE
Added adaptation of res50 and seuzeedet_plus to NEXAREAR data

### PR DESCRIPTION
**Pull request by srippa-nexar**

### CODE REVIEW

File: scripts/train.sh [hunk 1]
```diff
@@ -51,42 +51,28 @@ fi
...
   --image_set=train \
-  --train_dir=/tmp/bichen/logs/SqueezeDetPlus/train \
+  --train_dir=/opt/logs/NEXAREAR/SqueezeDetPlus/train  \
   --net=squeezeDet+ \
   --summary_step=100 \
   --checkpoint_step=500 \
   --gpu=$USE_GPU
 fi
 
-if [[ "$1" == "vgg16" ]]
```
Comment (roiaddi) [inline scripts/train.sh:65]: like

File: scripts/train.sh [hunk 2]
```diff
@@ -10,7 +10,7 @@ fi
 while test $# -gt 0; do
```
Comment (roiaddi) [inline scripts/train.sh:10]: when we have model class this script will have to go as well

File: src/config/nexarear_squeezeDetPlus_config.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+  mc.PERCENTAGE_OF_TEST_SET = 0.3
+  mc.RANDOM_SEED_TEST_TRAIN_SPLIT = 314
+
+  return mc
+
+def set_anchors(mc):
+  H, W, B = 43, 78, 9
+  anchor_shapes = np.reshape(
+      [np.array(
```
Comment (roiaddi) [inline src/config/nexarear_squeezeDetPlus_config.py:56]: can we calculate anchors based on the H and W?

File: src/config/nexarear_res50_config.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      np.transpose(
+          np.reshape(
+              np.array([np.arange(1, W+1)*float(mc.IMAGE_WIDTH)/(W+1)]*H*B),
+              (B, H, W)
+          ),
+          (1, 2, 0)
+      ),
+      (H, W, B, 1)
+  )
+  center_y = np.reshape(
```
Comment (roiaddi) [inline src/config/nexarear_res50_config.py:67]: duplicate code... please move what we can to common place

File: src/config/nexarear_vgg16_config.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
+# Author: Bichen Wu (bichen@berkeley.edu) 08/25/2016
+
+"""Model configuration for pascal dataset"""
```
Comment (roiaddi) [inline src/config/nexarear_vgg16_config.py]: if vgg is out, why do we need this class?

File: src/config/nexarear_squeezeDetPlus_config.py (hunk 3 above)
Comment (srippa-nexar) [reply src/config/nexarear_squeezeDetPlus_config.py:56]: Not really. The anchor boxes are the same for squeezeDet and squeezeDet+ but different for resnet

File: scripts/train.sh (hunk 2 above)
Comment (srippa-nexar) [reply scripts/train.sh:10]: It will go

File: src/config/nexarear_vgg16_config.py (hunk 5 above)
Comment (srippa-nexar) [reply src/config/nexarear_vgg16_config.py]: Removde

**Pull request merged**